src/Utils/FindUserByUserId.ts

```diff
@@ -0,0 +1,6 @@
+import { User } from '../Types/user';
+import usersFromServer from './../api/users';
```

Comment:
[CODE STYLE] - Don't use relative paths with '..' when importing modules. Use absolute paths based on the project's base directory to make imports cleaner and more manageable.